### PR TITLE
fix: version module to Go expectations

### PR DESCRIPTION
**Pull request by rawkode**
This will require a new v2 tag on the repository also

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package main
 
 import (
-	"github.com/go-gandi/terraform-provider-gandi/gandi"
+	"github.com/go-gandi/terraform-provider-gandi/gandi/v2"
```
Comment (nlewo) [inline main.go:4]:
```suggestion
	"github.com/go-gandi/terraform-provider-gandi/v2/gandi"
```

**Comment by nlewo** [comment]
FYI, there have been some works related to Pulumi made by @vincentbernat: https://github.com/vincentbernat/pulumi-gandi

**Comment by rawkode** [comment]
Ah, I added one to Pulumiverse

https://github.com/pulumiverse/pulumi-gandi

**Comment by vincentbernat** [comment]
Sorry, I should have moved it to go-gandi, but I don't remember why I didn't do it (except lack of time). I definitely plan to do it.

**Comment by nlewo** [comment]
> dunno what's the best place for people to discover it?
 
@rawkode any idea?

**Pull request merged**